### PR TITLE
Add Mozi Klub and Sorozat Klub to musor.tv.channels.xml

### PR DESCRIPTION
**Pull request by adrianmihalko**

### CODE REVIEW

File: sites/musor.tv/musor.tv.channels.xml [hunk 1]
```diff
@@ -124,6 +124,7 @@
   <channel site="musor.tv" lang="hu" xmltv_id="MezzoLive.fr" site_id="MEZZOHD">Mezzo Live HD</channel>
   <channel site="musor.tv" lang="hu" xmltv_id="Minimax.hu" site_id="MINIMAX">Minimax</channel>
   <channel site="musor.tv" lang="hu" xmltv_id="MoziPlus.hu" site_id="MOZI_PLUSZ">Mozi+ (HD)</channel>
+  <channel site="musor.tv" lang="hu" xmltv_id="MoziKlub.hu" site_id="MOZI_KLUB">Mozi Klub</channel>
```
Comment (freearhey) [inline sites/musor.tv/musor.tv.channels.xml:127]: Invalid xmltv_id.

File: sites/musor.tv/musor.tv.channels.xml [hunk 2]
```diff
@@ -143,6 +144,7 @@
   <channel site="musor.tv" lang="hu" xmltv_id="RTLKetto.hu" site_id="RTL2">RTL KETTŐ (HD)</channel>
   <channel site="musor.tv" lang="hu" xmltv_id="SlagerTV.hu" site_id="SLAGERTV">Sláger TV (HD)</channel>
   <channel site="musor.tv" lang="hu" xmltv_id="SorozatPlus.hu" site_id="SOROZAT">Sorozat+</channel>
+  <channel site="musor.tv" lang="hu" xmltv_id="SorozatKlub.hu" site_id="SOROZAT_KLUB">Sorozat Klub</channel>
```
Comment (freearhey) [inline sites/musor.tv/musor.tv.channels.xml:147]: Invalid xmltv_id.

**Review by freearhey — CHANGES_REQUESTED** [review]
https://github.com/iptv-org/epg/blob/master/CONTRIBUTING.md#how-to-add-a-channel-to-the-guide

**Comment by adrianmihalko** [comment]
Removed invalid xmltv_id.

**Review by freearhey — APPROVED** [review]

**Pull request merged**